### PR TITLE
PWA queries

### PR DESCRIPTION
**Pull request by jrharalson**
Progress on #92

### CODE REVIEW

File: sql/2019/11_PWA/11_02.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#
+#   `httparchive.almanac.lighthouse_mobile_1k` archive = 245 MB
+#   `httparchive.lighthouse.2019_07_01_mobile` = 1.22 TB 
+
+SELECT
+    COUNT(url) AS total,
+    COUNTIF(CAST(JSON_EXTRACT_SCALAR(report, '$.audits.installable-manifest.score') as NUMERIC) = 1) AS score_sum,
+    ROUND(COUNTIF(CAST(JSON_EXTRACT_SCALAR(report, '$.audits.installable-manifest.score') as NUMERIC) = 1) * 100 / COUNT(url), 2) as score_percentage
+FROM
+    `httparchive.almanac.lighthouse_mobile_1k`
```
Comment (rviscomi) [inline sql/2019/11_PWA/11_02.sql]: Query the full LH data, not sample set.

```suggestion
    `httparchive.lighthouse.2019_07_01_mobile`
```

Comment (jrharalson) [reply sql/2019/11_PWA/11_02.sql]: Updated

**Comment by rviscomi** [comment]
Is this ready for another review?

**Comment by rviscomi** [comment]
@jrharalson have you had time to work on this?

File: sql/2019/11_PWA/11_02.sql (hunk 1 above)
Comment (jrharalson) [reply sql/2019/11_PWA/11_02.sql]: Moved to July table

File: sql/2019/11_PWA/11_01b.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#standardSQL
```
[the remark this answers is not in the document]
Comment (jrharalson) [reply sql/2019/11_PWA/11_01b.sql]: Moved to blink_features table.

**Comment by jrharalson** [comment]
Committed these updates. blink_feature table is very helpful for sure.

**Comment by jrharalson** [comment]
I havent been able to work through the larger PWA temp tables driving the other queries. I might need some help in generating them - I must be doing something wrong in executing them.

**Comment by jrharalson** [comment]
Ready for review - Havent been able to tackle several queries in this Chapter though

**Pull request merged**